### PR TITLE
Added table

### PR DESCRIPTION
**Pull request by hrischuk-da**

### CODE REVIEW

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 1]
```diff
@@ -83,7 +83,140 @@ are:
 
 -  A new interface instance can be added to a template.
 
-Please consult the detailed documentation below for more information.
+The following table summarizes the changes that SCU supports.  Please consult
+the detailed documentation below for additional information.
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:87]:
```suggestion
The following table summarizes the changes supported by that SCU. Consult
the detailed documentation below for additional information.
```
Add link to "below"?

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 2]
```diff
@@ -83,7 +83,140 @@ are:
...
+|                 |               |            | otherwise the contracts cannot be |
+|                 |               |            | used by an upgraded package (this |
+|                 |               |            | fails at runtime).                |
+|                 |               |            |                                   |
+|                 |               |            | One can change the function that  |
+|                 |               |            | computes the signatories (for     |
+|                 |               |            | example to take into account new  |
+|                 |               |            | fields) when creating contracts.  |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | The type of a key cannot be       |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:121]: This is awkward and needs to be rephrased - perhaps something like "The key's type cannot be changed - for example, you cannot add a new field to a record that the key contains."

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 3]
```diff
@@ -83,7 +83,140 @@ are:
...
+|                 |               |            | the function computing the        |
+|                 |               |            | signatories can be changed. A     |
+|                 |               |            | persisted contract can be used by |
+|                 |               |            | an upgrade only if the result of  |
+|                 |               |            | the function does not change.     |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | Same as template signatories      |
+| Observers       | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of key   | Template      | No         | Same as template signatories      |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:136]:
```suggestion
| Change of Key   | Template      | No         | Same as template signatories      |
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 4]
```diff
@@ -83,7 +83,140 @@ are:
...
+| maintainers     | Definition    |            |                                   |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:137]:
```suggestion
| Maintainers     | Definition    |            |                                   |
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 5]
```diff
@@ -83,7 +83,140 @@ are:
...
+| Ensure          | Definition    |            | recalculated just before usage so |
+| Predicate       |               |            | any changes need to ensure that a |
+|                 |               |            | V1 contract is still valid as a   |
+|                 |               |            | V2 contract.                      |
++-----------------+---------------+------------+-----------------------------------+
+| Addition of     | Template      | Yes, but   |                                   |
+| new Definition  | Definition    | they they  |                                   |
+| fields, enum,   |               | must be    |                                   |
+| and variant     |               | optional   |                                   |
+| constructors    |               |            |                                   |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:150]:
```suggestion
| Addition of     | Template      | Yes, but   |                                   |
| New Definition  | Definition    | they they  |                                   |
| Fields, Enum,   |               | must be    |                                   |
| and Variant     |               | optional   |                                   |
| Constructors    |               |            |                                   |
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 6]
```diff
@@ -83,7 +83,140 @@ are:
...
++-----------------+---------------+------------+-----------------------------------+
+| Deletion of     | Template      | No         |                                   |
+| fields from     | Definition    |            |                                   |
+| template        |               |            |                                   |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:154]:
```suggestion
| Deletion of     | Template      | No         |                                   |
| Fields from     | Definition    |            |                                   |
| Template        |               |            |                                   |
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 7]
```diff
@@ -83,7 +83,140 @@ are:
...
+| Deletion of     | Choice        | No         |                                   |
+| Choice          |               |            |                                   |
+| Parameters      |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Choice        | No         |                                   |
+| Consuming       |               |            |                                   |
+| choice to       |               |            |                                   |
+| Non-Consuming   |               |            |                                   |
+| Choice, or      |               |            |                                   |
+| vice-versa      |               |            |                                   |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:189]:
```suggestion
| Change of       | Choice        | No         |                                   |
| Consuming       |               |            |                                   |
| Choice to       |               |            |                                   |
| Non-Consuming   |               |            |                                   |
| Choice, or      |               |            |                                   |
| Vice-versa      |               |            |                                   |
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 8]
```diff
@@ -83,7 +83,140 @@ are:
...
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Choice        | Yes        |                                   |
+| Choice Body     |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Choice        | Yes,       | You can change a user data type   |
+| Choice return   |               | except for | (record, variant, enum) contained |
+| type            |               | tuple or   | in the return type. In this case  |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:196]:
```suggestion
| Change of       | Choice        | Yes,       | You can change a user data type   |
| Choice Return   |               | except for | (record, variant, enum) contained |
| Type            |               | tuple or   | in the return type. In this case  |
```

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 9]
```diff
@@ -83,7 +83,140 @@ are:
...
+|                 |               | scalar     | you can add a field to the record |
+|                 |               | types      | or a new case to the              |
+|                 |               |            | variant/enum.                     |
+|                 |               |            |                                   |
+|                 |               |            | In particular: you cannot change  |
+|                 |               |            | a collection, Optional, List, or  |
+|                 |               |            | Map; you can change Tuple, (to a  |
+|                 |               |            | triple) if you define your own    |
+|                 |               |            | type.                             |
+|                 |               |            |                                   |
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:206]: Make capitalization consistent in this list

**Review by carrielaben-da — APPROVED** [review]
Small changes.

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst (hunk 2 above)
Comment (hrischuk-da) [reply docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst]: Change

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst (hunk 9 above)
Comment (hrischuk-da) [reply docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:206]: Changed

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 10]
```diff
@@ -83,7 +83,140 @@ are:
...
+|                 |               |            | otherwise the contracts cannot be |
+|                 |               |            | used by an upgraded package (this |
+|                 |               |            | fails at runtime).                |
+|                 |               |            |                                   |
+|                 |               |            | One can change the function that  |
+|                 |               |            | computes the signatories (for     |
+|                 |               |            | example to take into account new  |
+|                 |               |            | fields) when creating contracts.  |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | The key's type cannot be changed, |
```
Comment (samuel-williams-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:121]: Should this not be "Change Of Key"?

File: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst [hunk 11]
```diff
@@ -83,7 +83,140 @@ are:
...
+| Field Type      | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field           | Definition    |            |                                   |
+| Ordering        |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field Name      | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | A new template version needs to   |
```
Comment (samuel-williams-da) [inline docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst:109]: Unsure how I feel about marking this as simply "Not covered by SCU". We have special logic to allow these to change with new contracts, and to recompute previous values.
Perhaps "Covered by SCU" isn't the right phrasing here, but instead something akin to "Can change in an upgrade"

**Pull request merged**